rllib/algorithms/impala/tf/impala_tf_learner.py

```diff
@@ -93,16 +95,17 @@ def compute_loss_per_module(
         mean_vf_loss = vf_loss / batch_size
 
         # The entropy loss.
-        entropy_loss = -tf.reduce_sum(target_actions_logp_time_major)
+        mean_entropy_loss = -tf.reduce_mean(target_policy_dist.entropy())
```

Comment:
We should use mean (over sum)! Just like we do (and have been doing) for APPO. Using the sum will add additional dependency (and the need to adjust the coeff) on the number of actions, I believe.

Reply:
Note that this only affects the new stack, so old benchmarks remain valid.